aea/cli_gui/aea_cli_rest.yaml

```diff
@@ -0,0 +1,99 @@
+openapi: "3.0.2"
```

Comment:
I noticed that `swagger.yaml` is the one loaded in `app.py`. What's the purpose of this OpenAPI file?